### PR TITLE
Fix part of #219: update dft filesnames by running part of data-raw/misc.Rmd

### PR DESCRIPTION
**Pull request by layik**
Ran the https://github.com/ropensci/stats19/blob/master/data-raw/misc.Rmd file part related to updating file names. Nothing else.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Amazing, looks good to me Layik!

File: data-raw/file_name_df.csv [hunk 1]
```diff
@@ -12,12 +12,26 @@ BloodAlcoholContent-CoronersMatchedData2009.zip,https://data.dft.gov.uk/road-acc
 DigitalBreathTestData2011.zip,https://data.dft.gov.uk/road-accidents-safety-data/DigitalBreathTestData2011.zip
 BloodAlcoholContent-CoronersMatchedData2010.zip,https://data.dft.gov.uk/road-accidents-safety-data/BloodAlcoholContent-CoronersMatchedData2010.zip
 DigitalBreathTestData2010.zip,https://data.dft.gov.uk/road-accidents-safety-data/DigitalBreathTestData2010.zip
+dft-road-casualty-statistics-vehicle-e-scooter-provisional-mid-year-unvalidated-2022.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-vehicle-e-scooter-provisional-mid-year-unvalidated-2022.csv
```
Comment (Robinlovelace) [inline data-raw/file_name_df.csv:15]: :+1:

File: data-raw/file_name_df.csv [hunk 2]
```diff
@@ -12,12 +12,26 @@ BloodAlcoholContent-CoronersMatchedData2009.zip,https://data.dft.gov.uk/road-acc
...
+dft-road-casualty-statistics-casualty-provisional-mid-year-unvalidated-2021.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-casualty-provisional-mid-year-unvalidated-2021.csv
+dft-road-casualty-statistics-vehicle-e-scooter-provisional-mid-year-unvalidated-2021.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-vehicle-e-scooter-provisional-mid-year-unvalidated-2021.csv
 dft-road-casualty-statistics-casualty-2020.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-casualty-2020.csv
 dft-road-casualty-statistics-vehicle-2020.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-vehicle-2020.csv
 dft-road-casualty-statistics-accident-2020.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-accident-2020.csv
-dft-road-casualty-statistics-casualty-1979-2020.csv,https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-casualty-1979-2020.csv
```
Comment (Robinlovelace) [inline data-raw/file_name_df.csv:18]: Goodbye old deleted data!

**Review by Robinlovelace — APPROVED** [review]

**Comment by layik** [comment]
Thank you @Robinlovelace

**Pull request merged**